language/move-lang/src/parser/syntax.rs

```diff
@@ -1360,6 +1360,50 @@ fn parse_field_annot<'input>(tokens: &mut Lexer<'input>) -> Result<(Field, Type)
     Ok((f, st))
 }
 
+//**************************************************************************************************
+// Address Block
+//**************************************************************************************************
+
+// Parse an address block:
+//      Address Block =
```

Comment:
Minor nit: for consistency can you drop the whitespace and make it "AddressBlock"? Also in the comment before parse_file()